doc/source/tokudb/toku_backup.rst

```diff
@@ -151,20 +156,48 @@ Reporting Errors
 
- mysql> SELECT @@tokudb_backup_last_error;
- +----------------------------+
- | @@tokudb_backup_last_error |
- +----------------------------+
- |                         17 |
- +----------------------------+
+  mysql> SELECT @@tokudb_backup_last_error;
+  +----------------------------+
+  | @@tokudb_backup_last_error |
+  +----------------------------+
+  |                         17 |
+  +----------------------------+
  
- mysql> SELECT @@tokudb_backup_last_error_string;
- +---------------------------------------------------+
- | @@tokudb_backup_last_error_string                 |
- +---------------------------------------------------+
- | tokudb backup couldn't create needed directories. |
- +---------------------------------------------------+
+  mysql> SELECT @@tokudb_backup_last_error_string;
+  +---------------------------------------------------+
+  | @@tokudb_backup_last_error_string                 |
+  +---------------------------------------------------+
+  | tokudb backup couldn't create needed directories. |
+  +---------------------------------------------------+
+
+Limitations and known issues
+----------------------------
+
+* You must disable |InnoDB| asynchronous IO if backing up |InnoDB| tables with |TokuBackup|. Otherwise you will have inconsistent, unrecoverable backups. The appropriate setting is ``innodb_use_native_aio=0``.
+  
+* To be able to run Point-In-Time-Recovery you'll need to manually get the binary log position.
+
+* Transactional storage engines (|TokuDB| and |InnoDB|) will perform recovery on the backup copy of the database when it is first started.
+
+* Tables using non-transactional storage engines (|MyISAM|) are not locked during the copy and may report issues when starting up the backup. It is best to avoid operations that modify these tables at the end of a hot backup operation (adding/changing users, stored procedures, etc.).
+
+* The database is copied locally to the path specified in :file:`/path/to/backup`. This folder must exist, be writable, be empty, and contain enough space for a full copy of the database.
+
+* |TokuBackup| always makes a backup of the |MySQL| :variable:`datadir` and optionally the :variable:`tokudb_data_dir`, :variable:`tokudb_log_dir`, and the binary log folder. The latter three are only backed up separately if they are not the same as or contained in the |MySQL| :variable:`datadir`. None of these three folders can be a parent of the |MySQL| :variable:`datadir`.
+
+* No other directory structures are supported. All |InnoDB|, |MyISAM|, and other storage engine files must be within the |MySQL| :variable:`datadir`.
+
+* |TokuBackup| does not follow symbolic links.
+
+* |TokuBackup| does not backup |MySQL| configuration file(s). 
+
+* |TokuBackup| does not backup tablespaces if they are out of :variable:`datadir`.
+
+* |TokuBackup| can't recover backed-up table data if backup was taken while running ``OPTIMIZE TABLE`` or ``ALTER TABLE ... TABLESPACE``
```

Comment:
Might want to refer to the upstream and/or Percona issues that indicate the cause of the problem.